pandas-stubs/core/groupby/groupby.pyi

```diff
@@ -7,7 +7,6 @@ from pandas.core.base import (
 from pandas.core.frame import DataFrame
 from pandas.core.generic import NDFrame
 from pandas.core.groupby import ops
-from pandas.core.groupby.indexing import GroupByIndexingMixin
```

Comment:
not defined in the stubs - but it provides only private methods, so I removed it.